app/mailers/report_mailer.rb

```diff
@@ -14,6 +14,32 @@ def budget_status
     )
   end
 
+  def annual_report(year, to_email)
+    proposals = Proposal.approved.where(client_data_type: "Ncr::WorkOrder")
```

Comment:
since this appears to be an `ncr`-specific mailer, should we include `ncr` in the mailer method name?